.gitignore

```diff
@@ -2,3 +2,5 @@
 .Rhistory
 .RData
 .Ruserdata
+figure_1.png
+model_RMSEs.csv
```

Comment:
Let's go ahead and specify that anything in `1_fetch/out` or `2_process/out` is ignored by git. You can do so by adding the line `*/out/*` to the `.gitignore` file. Since git is already tracking those files, you'll need to tell it to [stop tracking](https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign) them with `git rm --cached <file>`.

Reply:
And if you add `*/out/*` you'll no longer need to list `figure_1.png` and `model_RMSEs.csv`